doc-content/enterprise-only/vehicle-route-planning/run-vrp-manually-proc.adoc

```diff
@@ -0,0 +1,76 @@
+= Configure and run OptaWeb Vehicle Routing manually
+
+The easiest way to run OptaWeb Vehicle Routing is to use the `runlocally.sh` script. However, if Bash is not available on your system you can manually complete the steps that the `runlocally.sh` script performs.
+
+.Prerequisites
+* OptaWeb Vehicle Routing has been successfully built with Maven as described in xref:download-ref-imp-proc_{context}[].
+* Internet access is available.
+
+.Procedure
+. Download routing data.
++
+The routing engine requires geographical data to calculate the time it takes vehicles to travel between locations.
+You must download and store OpenStreetMap (OSM) data files on the local file system before you run OptaWeb Vehicle Routing.
++
+NOTE: The OSM data files are typically between 100 MB to 1 GB and take time to download so it is a good idea to download the files before building or starting the OptaWeb Vehicle Routing application.
+
+.. Open `http://download.geofabrik.de/` in a web browser.
+.. Click a region in the *Sub Region* list, for example *Europe*. The sub region's page opens.
+.. In the *Sub Regions* table, download the OSM file (`.osm.pbf`) for a country, for example Belgium.
+
+. Create the data directory structure.
++
+OptaWeb Vehicle Routing reads and writes several types of data on the file system.
+It reads OSM (OpenStreetMap) files from the `openstreetmap` directory, writes a road network graph to the `graphhopper` directory, and persists user data in a directory called `db`.
+Create a new directory dedicated to storing all of these data to make it easier to upgrade to a newer version of OptaWeb Vehicle Routing in the future and continue working with the data you created previously.
+
+.. Create the `openstreetmap` directory in your user account `home` directory, for example:
++
+[source,subs="attributes+"]
+----
+$HOME/{data-dir-name}
+└── openstreetmap
+----
+
+.. Move all of your downloaded OSM files (files with the extension `.osm.pbf`) to the `openstreetmap` directory.
++
+The rest of the directory structure will be created by the OptaWeb Vehicle Routing application when it runs for the first time.
+After that, your directory structure will look similar to the following example:
++
+[source,subs="attributes+"]
+----
+$HOME/{data-dir-name}
+├── db
+│   └── vrp.mv.db
+├── graphhopper
+│   └── belgium-latest
+└── openstreetmap
+    └── belgium-latest.osm.pbf
+----
+// TODO maybe replace this with a screenshot, doesn't look good in PDF.
+. To run OptaWeb Vehicle Routing, enter the following command:
++
+[source,subs="attributes+"]
+----
+java -jar optaweb-vehicle-routing-standalone-{PRODUCT_VERSION}.jar \
+--app.persistence.h2-dir=$HOME/<DATA_DIR_NAME>/db \
+--app.routing.gh-dir=$HOME/<DATA_DIR_NAME>/graphhopper
+--app.routing.osm-dir=$HOME/<DATA_DIR_NAME>/openstreetmap \
+--app.routing.osm-file=<COUNTRY>-latest.osm.pbf \
+--app.region.country-codes=<REGION_CODE>
+----
+In this command, replace the following variables:
```

Comment:
For example what works for me is:
java -jar optaweb-vehicle-routing-standalone-7.39.0.Final-redhat-00001.jar --app.persistence.h2.dir=/home/adupliak/VRPDataDir/db --app.routing.gh-dir=/home/adupliak/VRPDataDir/graphhopper --app.routing.osm-dir=~/VRPDataDir/openstreetmap --app.routing.osm-file=/home/adupliak/VRPDataDir/openstreetmap/northwestern-fed-district-latest.osm.pbf --app.region.country-codes=RU

Reply:
Thanks. Add the example.